wrappers/python/simtk/openmm/app/internal/customgbforces.py

```diff
@@ -29,19 +29,26 @@
 USE OR OTHER DEALINGS IN THE SOFTWARE.
 """
 
+
 from __future__ import division, absolute_import
 
 from collections import defaultdict
 import copy
+import numpy as np
+from scipy import interpolate
```

Comment:
OpenMM doesn't require numpy or scipy, so you can't assume they'll be available.  That's why the test cases are failing.